pkg/storage/legacysql/dualwrite/kvstore.go

```diff
@@ -0,0 +1,83 @@
+package dualwrite
+
+import (
+	"context"
+	"encoding/json"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+
+	"github.com/grafana/grafana-app-sdk/logging"
+	"github.com/grafana/grafana/pkg/infra/kvstore"
+)
+
+type keyvalueDB struct {
+	db     kvstore.KVStore
+	logger logging.Logger
+}
+
+const globalKVOrgID = 0
```

Comment:
can you explain how this will work and the limitations of this dualwriting store?